test_holiday_functions/test_purple_holiday_functions.py

```diff
@@ -14,6 +14,8 @@
 
 # Anabelle
 
+assert not naughty_or_nice(5,9)
+assert not brings_figgy_pudding(2,3,12,13)
```

Comment:
Just some whitespace adjustments

```suggestion
assert not naughty_or_nice(5, 9)
assert not brings_figgy_pudding(2, 3, 12, 13)
```